website/docs/Pipelines/Multiome_Pipeline/README.md

```diff
@@ -102,7 +102,8 @@ The Multiome workflow calls two WARP subworkflows, one external subworkflow (opt
 |--- | --- | --- | 
 | multiome_pipeline_version_out | N.A. | String describing the version of the Multiome pipeline used. |
 | bam_aligned_output_atac | `<input_id>_atac.bam` | BAM file containing aligned reads from ATAC workflow. |
-| fragment_file_atac | `<input_id>_atac.fragments.tsv` | TSV file containing fragment start and stop coordinates per barcode. | 
+| fragment_file_atac | `<input_id>_atac.fragments.sorted.tsv.gz` | A sorted and bgzipped TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "ATAC Barcode", "Number of reads", and "GEX Barcode". | 
```

Comment:
```suggestion
| fragment_file_atac | `<input_id>_atac.fragments.sorted.tsv.gz` | Sorted and bgzipped TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "ATAC Barcode", "Number of reads", and "GEX Barcode". | 
```

Reply:
Should the columns here be the same as the columns listed in the ATAC Overview doc?